### ISSUE TITLE
Provide a way to ignore GET query params

### ISSUE THREAD

**Issue by kontinuity** [ozantunca/elb-log-analyzer#17]
We daily process our ALB logs to find the most popular queries however with slight variations or per user GET params, the numbers are askew.

e.g.
```
https://example.com/user/list?req_by=userA
https://example.com/user/list?req_by=userB
```

We'd like to ignore the query params here.

**Comment by ozantunca** [comment]
Makes sense to me. I'll look into it and update here

**Comment by ozantunca** [comment]
So I'm thinking that the right way to handle this could be to add some new virtual fields. Right now, to get the most popular URLs, one would need to use `requested_resource` field which would show a full URL much like this: `https://example.com:80/user/list?req_by=userA`. Perhaps we can parse this URL and provide its pieces as separate fields. Then we can have additional fields such as:

`requested_resource.pathname`: `/user/list`
`requested_resource.host`: `example.com`
`requested_resource.protocol`: `https`
`requested_resource.port`: `80`
`requested_resource.search`: `?req_by=userA`

Would that solve your issue?

**Comment by ozantunca** [comment]
These changes are included in v1.3
